src/os/spawn.php

```diff
@@ -0,0 +1,173 @@
+<?hh
+
+namespace HH\Lib\Experimental\OS;
+use namespace HH\Lib\C;
+use namespace HH\Lib\Vec;
+use namespace HH\Lib\_Private\_OS;
+use type HH\Lib\OS\{FileDescriptor, IsNotADirectoryException};
+use function HH\Lib\OS\{close, open};
```

Comment:
Remove both `use type` and `use function`; add `use namespace HH\Lib\OS`

Change references to FileDescriptor/IsNotADirectory/close/open throughout.

This:
- makes it clearer to read
- keeps them `grep`pable